CHANGES.rst

```diff
@@ -1,4 +1,4 @@
-Changes to be released in next version
+Changes in 0.18.4 (2021-03-03)
```

Comment:
I think we don't need to (maybe must not) change this line, as it'll be updated by our scripts.